test/linter/test-browsers-presence.js

```diff
@@ -28,12 +28,15 @@ const browsers = {
 };
 
 /**
- * @param {Identifier} data
- * @param {string[]} displayBrowsers
- * @param {string[]} requiredBrowsers
- * @param {string} category
- * @param {Logger} logger
- * @param {string} [path]
+ * Check the data for any disallowed browsers or if it's missing required browsers
+ *
+ * @param {Identifier} data The data to test
+ * @param {string[]} displayBrowsers All of the allowed browsers for this data
+ * @param {string[]} requiredBrowsers All of the required browsers for this data
+ * @param {string} category The category the data belongs to
+ * @param {Logger} logger The logger to output errors to
+ * @param {string} [path] The path of the data
```

Comment:
In general, the full-stop would also not hurt after param/property/returns descriptions:

```suggestion
 * @param {string[]} displayBrowsers All of the allowed browsers for this data.
 * @param {string[]} requiredBrowsers All of the required browsers for this data.
 * @param {string} category The category the data belongs to.
 * @param {Logger} logger The logger to output errors to.
 * @param {string} [path] The path of the data.
```

PS: I noticed that https://jsdoc.app/tags-param.html ends descriptions with full-stops as well.